ui/src/app/views/concept-table/component.tsx

```diff
@@ -0,0 +1,140 @@
+import {Component, EventEmitter, Input, Output} from '@angular/core';
+import {Clickable} from 'app/components/buttons';
+import {CheckBox} from 'app/components/inputs';
+import {PopupTrigger} from 'app/components/popups';
+import {ReactWrapperBase, toggleIncludes} from 'app/utils';
+import {reactStyles} from 'app/utils';
+import {Concept} from 'generated/fetch/api';
+import * as fp from 'lodash/fp';
+import {Column} from 'primereact/column';
+import {DataTable} from 'primereact/datatable';
+import * as React from 'react';
```

Comment:
nit: per convention we've been separating out local imports from package imports, so i would put react, fp, angular, and primereact in their own block